### PR TITLE
[Doc] Readme Update

### PR DESCRIPTION
**Pull request by jpeng-ms**
## Purpose
updated readme for chat initial release

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## Pull Request Type
What kind of change does this Pull Request introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [x] Documentation content changes
- [ ] Other... Please describe:

## Pull request checklist

This PR has considered:
- [ ] Color Theming
- [ ] iOS supported versions (all major versions greater than or equal current target deployment version)
- [ ] VoiceOver and Keyboard Accessibility
- [ ] Internationalization and Right to Left layouts
- [ ] Size classes and window sizes (iPhone vs iPad, notched devices, multitasking, different window sizes, etc)
- [ ] View Data Injection
- [ ] Demo App UI/UX update

## How to Test

* Open...
* Click on...

## What to Check

| Before                                       | After                                      |
|----------------------------------------------|--------------------------------------------|
| Screenshot or description before this change | Screenshot or description with this change |

### CODE REVIEW

File: AzureCommunicationUI/sdk/AzureCommunicationUIChat/README.md [hunk 1]
```diff
@@ -10,6 +10,10 @@
 
 - Public Preview: [AzureCommunicationUIChat_1.0.0-beta.1](https://github.com/Azure/communication-ui-library-ios/releases/tag/AzureCommunicationUIChat_1.0.0-beta.1)
 
+## Getting Started
+
+Get started with Azure Communication Services by using the UI Library to integrate communication experiences into your applications. For detailed instructions to quickly integrate the UI Library functionalities visit the [Quick-start Documentation](https://docs.microsoft.com/en-us/azure/communication-services/quickstarts/ui-library/get-started-chat-ui-library?tabs=kotlin&pivots=platform-ios).
```
Comment (palatter) [inline AzureCommunicationUI/sdk/AzureCommunicationUIChat/README.md:15]: Is the tabs=kotlin part of the link supposed to be here?

**Comment by JoshuaLai** [comment]
I think it should be fine to merge this into develop. It will be easier to synchronize once its in develop and when the docs are officially out on release day we will merge this into main without accidentally forgetting about it

**Comment by jpeng-ms** [comment]
> I think it should be fine to merge this into develop. It will be easier to synchronize once its in develop and when the docs are officially out on release day we will merge this into main without accidentally forgetting about it

I think it might be okay. I can rebase this to release branch later on. I will keep this draft open so we won't miss it

File: AzureCommunicationUI/sdk/AzureCommunicationUICalling/README.md [hunk 2]
```diff
@@ -1,14 +1,19 @@
...
+
 ![Cocoapods](https://img.shields.io/cocoapods/l/AzureCommunicationUICalling)
 ![CocoaPods Compatible](https://img.shields.io/cocoapods/v/AzureCommunicationUICalling)
 ![Cocoapods platforms](https://img.shields.io/cocoapods/p/AzureCommunicationUICalling)
 
 ## Latest Release
 
-- [1.1.0 release](https://github.com/Azure/communication-ui-library-ios/releases/tag/AzureCommunicationUICalling_1.1.0)
+- Stable: [1.1.0 release](https://github.com/Azure/communication-ui-library-ios/releases/tag/AzureCommunicationUICalling_1.1.0)
+- Public Preview: [1.2.0-beta.1](https://github.com/Azure/communication-ui-library-ios/releases/tag/AzureCommunicationUICalling_1.2.0-beta.1)
```
Comment (emlynmac) [inline AzureCommunicationUI/sdk/AzureCommunicationUICalling/README.md:12]: Why are we bumping the calling release to 1.2 beta, on the chat release branch?

File: AzureCommunicationUI/sdk/AzureCommunicationUICalling/manual-installation.md [hunk 3]
```diff
@@ -1,17 +1,17 @@
 # Embedded Framework Installation
 
-It is possible to add the UI library manually, rather than using cocoapods.
+This documentation is meant to facilitate developers who wish to embed UI library manually without using Cocoapods. Since UI library contains a couple of dependencies, you would need to embed those manually as well.
```
Comment (emlynmac) [inline AzureCommunicationUI/sdk/AzureCommunicationUICalling/manual-installation.md:3]: We probably should keep the UI Calling documentation updates away from the chat release branch

**Review by emlynmac — COMMENTED** [review]
The chat release branch should have only chat updates.
Let's create a separate PR for the changes to the calling instructions

**Review by vhuseinova-msft — DISMISSED** [review]

File: AzureCommunicationUI/sdk/AzureCommunicationUICalling/README.md (hunk 2 above)
Comment (jpeng-ms) [reply AzureCommunicationUI/sdk/AzureCommunicationUICalling/README.md:12]: 1.2.0 beta 1 has been released for 2 months now, we need to have doc updated to reflect that. This is true whether we are merging into release or not

File: AzureCommunicationUI/sdk/AzureCommunicationUICalling/manual-installation.md (hunk 3 above)
Comment (jpeng-ms) [reply AzureCommunicationUI/sdk/AzureCommunicationUICalling/manual-installation.md:3]: we have made release cut to chat already. Do we want to release incorrect instruction?

**Comment by jpeng-ms** [comment]
> The chat release branch should have only chat updates.
> Let's create a separate PR for the changes to the calling instructions

so the intention here is to update the readme that would be reflected on main branch

File: AzureCommunicationUI/sdk/AzureCommunicationUICalling/README.md (hunk 2 above)
Comment (emlynmac) [reply AzureCommunicationUI/sdk/AzureCommunicationUICalling/README.md:12]: The release information for calling does not belong on a chat release branch.
If this was missed previously, it needs to be added as a PR to the main branch as a hot fix for calling as it was forgotten there.
Chat release branch is about changes only required for releasing chat; nothing more

Comment (emlynmac) [reply AzureCommunicationUI/sdk/AzureCommunicationUICalling/README.md:12]: Following up from the parking lot discussion at standup today; we should not be calling out pre-releases in documentation. 
Let's remove line 12 and get chat released.

Comment (jpeng-ms) [reply AzureCommunicationUI/sdk/AzureCommunicationUICalling/README.md]: thanks updated

File: AzureCommunicationUI/sdk/AzureCommunicationUICalling/manual-installation.md (hunk 3 above)
Comment (jpeng-ms) [reply AzureCommunicationUI/sdk/AzureCommunicationUICalling/manual-installation.md]: as discussed offline, this change has been reverted

File: AzureCommunicationUI/sdk/AzureCommunicationUIChat/CHANGELOG.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
 # Release History
 
 
-## 0.1.0-beta.1 (TBA)
+## 1.0.0-beta.1 (Jan 10, 2023)
```
Comment (vhuseinova-msft) [inline AzureCommunicationUI/sdk/AzureCommunicationUIChat/CHANGELOG.md:4]: The date format isn't the same as in calling.
Calling uses 2022-12-14

Comment (jpeng-ms) [reply AzureCommunicationUI/sdk/AzureCommunicationUIChat/CHANGELOG.md]: Thanks updated

**Review by vhuseinova-msft — APPROVED** [review]

**Review by palatter — APPROVED** [review]

**Pull request merged**